W02SP.html

```diff
@@ -1,36 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Minimalist Portfolio</title>
+    <link rel="stylesheet" href="https://use.typekit.net/srs1oxj.css" />
+    <link rel="stylesheet" href="CSSwebsite.css" />
+    <div class="section-header" />
+  </head>
 
-<head>
-	<meta charset="UTF-8" />
-	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-	<title>Minimalist Portfolio</title>
-	<link rel="stylesheet" href="CSSwebsite.css" />
-	<link rel="stylesheet" href="https://use.typekit.net/srs1oxj.css">
-	<div class="section-header">
-	</div>
+  <!-- sections -->
+  <div class="section-header">
+    <div class="section-header-container">
+      <div class="section-title">
+        <a class="home_link" href="W02FP.html">Home</a>
+      </div>
+      <div class="section-order">/001</div>
+    </div>
+  </div>
+  <!-- <div class="whitespace"></div> -->
+  <div class="divider"></div>
+  <div class="menu__item">
+    <a class="menu__item-static">/Italy</a>
+  </div>
 
-	<!-- sections -->
-	<div class="section-header">
-		<div class="section-header-container">
-			<div class="section-title">
-				<a class="home_link" href="W02FP.html">Home</a>
-			</div>
-			<div class="section-order">/001
-			</div>
-		</div>
-	</div>
-	<!-- <div class="whitespace"></div> -->
-	<div class="divider"></div>
-	<div class="menu__item">
-		<a class="menu__item-static">/Italy</a>
-	</div>
-	</div>
-	</div>
-	</section>
-
-	<div class="myGallery">
-		<img src="ImageGrids.jpg" />
-		<
-
-	</div>
\ No newline at end of file
+  <div style="width: 100%" class="gallery">
+    <img style="width: 100vw" src="assets/gridimage.png" />
```

Comment:
@hannahstowell Here is where you will want to change the `src` of the image to be whatever your image file is named.